Library/RSBot.Core/Objects/InventoryItem.cs

```diff
@@ -343,5 +343,57 @@ public bool CanBeEquipped()
 
             return true;
         }
+
+        /// <summary>
+        /// Creates a clone from this.
+        /// </summary>
+        /// <returns>The clone <see cref="InventoryItem"/> instance.</returns>
+        public InventoryItem CreateClone()
```

Comment:
Did you consider implementing IClonable? Might be useful for other Classes, too. https://dotnetcoretutorials.com/2020/09/09/cloning-objects-in-c-and-net-core/ I like the idea of a CloningService which is called in the IClonable.Clone() method.

Reply:
I thought about it. I prefer the ICloneable<T> interface with the T Clone () method.